freud/order.pyx

```diff
@@ -213,6 +213,10 @@ cdef class Nematic(_Compute):
         orientations = freud.util._convert_array(
             orientations, shape=(None, 3))
 
+        if len(np.where(~orientations.any(axis=1))[0])!=0:
+            warnings.warn('You supplied a zero vector in the orientations'
+                          'array, which causes unusual behavior.',
```

Comment:
```suggestion
            warnings.warn('The zero vector was detected in the orientations array, which causes unusual behavior.',
```
To be clear, what exactly is the output when the zero vector is given as the input? I'd like to change the message to be a bit more explicit about what the user should expect to see.

Reply:
`op.order` is zero; Entries of `op.nematic_tensor` are `NaN`; `op.director` points in the wrong direction. However `op.particle_tensors` are correct. I agree a more explicit error message would be better but I don't think fitting in all these details is necessary.

Reply:
Let's use the following error message then:

'Including zero vector in the orientations array may lead to undefined behavior.'